release-content/migration-guides/convert-coordinates.md

```diff
@@ -79,10 +98,9 @@ let handle = asset_server.load_with_settings(
 );
 ```
 
-After opting into the new behavior, your scene will be oriented such that your modeling software's forward direction correctly corresponds to Bevy's forward direction.
+After opting into the new behavior, your scene will be oriented such that other software's model forward direction correctly corresponds to Bevy's forward direction.
```

Comment:
This should mention that is only relevant to importing scene's through bevy's gLTF importer. If someone uses something else like FBX or obj we can't make that guarantee since they would use an external importer.